components/mitsubishi_uart/muart_utils.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "mitsubishi_uart.h"
+
+namespace esphome {
+namespace mitsubishi_uart {
+
+class MUARTUtils {
```

Comment:
Could we get a quick descriptive comment for each of these describing what they do?  Like I sorta understand what's happening, but it's not eminently clear.